File: TechEvents/src/LocataireBundle/Entity/Reservation.php
<?php

namespace LocataireBundle\Entity;
Use AppBundle\Entity\User;
use Doctrine\ORM\Mapping as ORM;

/**
 * Reservation
 *
 * @ORM\Table(name="reservation", indexes={@ORM\Index(name="fk_res_loc", columns={"id_local"})})
 * @ORM\Entity
 */
class Reservation
{
    /**
     *
     * @ORM\Column(name="id_reservation", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    protected $idReservation;

    /**
     * @var \User
     ** @ORM\ManyToOne(targetEntity="\UserBundle\Entity\User",inversedBy="id")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_owner", referencedColumnName="id")
     * })
     */
    private $idOwner;

    /**
     *
     *@var \User
     ** @ORM\ManyToOne(targetEntity="\UserBundle\Entity\User",inversedBy="id")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_user", referencedColumnName="id")
     * })*/
    private $idUser;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_debut", type="date", nullable=false)
     */
    private $dateDebut;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_fin", type="date", nullable=false)
     */
    private $dateFin;

    /**
     * @var \Local
     *
     * @ORM\ManyToOne(targetEntity="Local")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_local", referencedColumnName="ID_Loc")
     * })
     */
    private $idLocal;

    /**
     * @return int
     */
    public function getIdReservation()
    {
        return $this->idReservation;
    }

    /**
     * @param int $idReservation
     */
    public function setIdReservation($idReservation)
    {
        $this->idReservation = $idReservation;
    }

    /**
     * @return \User
     */
    public function getIdOwner()
    {
        return $this->idOwner;
    }

    /**
     * @param \User $idOwner
     */
    public function setIdOwner($idOwner)
    {
        $this->idOwner = $idOwner;
    }

    /**
     * @return \User
     */
    public function getIdUser()
    {
        return $this->idUser;
    }

    /**
     * @param \User $idUser
     */
    public function setIdUser($idUser)
    {
        $this->idUser = $idUser;
    }

    /**
     * @return \DateTime
     */
    public function getDateDebut()
    {
        return $this->dateDebut;
    }

    /**
     * @param \DateTime $dateDebut
     */
    public function setDateDebut($dateDebut)
    {
        $this->dateDebut = $dateDebut;
    }

    /**
     * @return \DateTime
     */
    public function getDateFin()
    {
        return $this->dateFin;
    }

    /**
     * @param \DateTime $dateFin
     */
    public function setDateFin($dateFin)
    {
        $this->dateFin = $dateFin;
    }

    /**
     * @return \Local
     */
    public function getIdLocal()
    {
        return $this->idLocal;
    }

    /**
     * @param \Local $idLocal
     */
    public function setIdLocal($idLocal)
    {
        $this->idLocal = $idLocal;
    }



    public function __toString()
    {
        return "stringtrong";
    }


}

File: TechEventsM/src/com/mycompany/myapp/entities/Reservation.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.myapp.entities;

import java.util.Date;


/**
 *
 * @author Brahim
 */
public class Reservation {
    private int idReservation;
private User idOwner;
private int idLoc;
private User idUser;
private Date dateDeb;
private Date dateFin;
    
    public Reservation(int idReservation, User idOwner, int idLoc, User idUser, Date dateDeb, Date dateFin) {
        this.idReservation = idReservation;
        this.idOwner = idOwner;
        this.idLoc = idLoc;
        this.idUser = idUser;
        this.dateDeb = dateDeb;
        this.dateFin = dateFin;
    }

    public Reservation(User idOwner, int idLoc,User idUser,Date dateDeb, Date dateFin) {
        this.idOwner = idOwner;
        this.idLoc = idLoc;
        this.idUser = idUser;
        this.dateDeb = dateDeb;
        this.dateFin = dateFin;
    }
    @Override
    public String toString() {
        return "ReservationLoc{" + "idReservation=" + idReservation + ", idOwner=" + idOwner + ", idLoc=" + idLoc + ", idUser=" + idUser + ", dateDeb=" + dateDeb + ", dateFin=" + dateFin + '}';
    }
    
    public int getIdReservation() {
        return idReservation;
    }

    public void setIdReservation(int idReservation) {
        this.idReservation = idReservation;
    }

    public User getIdOwner() {
        return idOwner;
    }

    public void setIdOwner(User idOwner) {
        this.idOwner = idOwner;
    }

    public int getIdLoc() {
        return idLoc;
    }

    public void setIdLoc(int idLoc) {
        this.idLoc = idLoc;
    }

    public User getIdUser() {
        return idUser;
    }

    public void setIdUser(User idUser) {
        this.idUser = idUser;
    }

    public Date getDateDeb() {
        return dateDeb;
    }

    public void setDateDeb(Date dateDeb) {
        this.dateDeb = dateDeb;
    }

    public Date getDateFin() {
        return dateFin;
    }

    public void setDateFin(Date dateFin) {
        this.dateFin = dateFin;
    }

}
